scratch-js/Sprite.mjs

```diff
@@ -609,4 +610,10 @@ export class Stage extends SpriteBase {
     // For obsolete counter blocks.
     this.__counter = 0;
   }
+
+  fireBackdropChanged() {
+      this._project.fireTrigger(Trigger.BACKDROP_CHANGED, {
+          backdrop: this.costume.name
+      });
+  }
```

Comment:
You're indenting with four spaces here, which is inconsistent with the rest of the project.

(I'll let you in on a cool trick, though. If you install the [Prettier extension](https://prettier.io/docs/en/editors.html) in your code editor, your formatting will be fixed automatically every time you save a file. Pretty nifty!)